src/fundus/scraping/crawler.py

```diff
@@ -209,7 +212,10 @@ def build_extraction_filter() -> Optional[ExtractionFilter]:
                         f"is(are) not supported by {publisher.publisher_name}. Skipping publisher"
                     )
                 else:
-                    fitting_publishers.append(publisher)
+                    if (not publisher.deprecated) or ignore_deprecated:
+                        fitting_publishers.append(publisher)
+                    else:
+                        logger.warning(f"Skipping deprecated publisher: {publisher.publisher_name}")
```

Comment:
Here's a logic bug included. If `only_complete` is set to `False`, the `ignore_deprecated` flag doesn't work anymore, because it is the `if isinstance(extraction_filter, Requires)` block.

Reply:
Thanks for cathing this